packages/wonder-blocks-modal/src/components/__docs__/modal-dialog.stories.js

```diff
@@ -0,0 +1,255 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import Color from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {
+    ModalLauncher,
+    ModalDialog,
+    ModalPanel,
+} from "@khanacademy/wonder-blocks-modal";
+import {Body} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+const customViewports = {
+    phone: {
+        name: "phone",
+        styles: {
+            width: "320px",
+            height: "568px",
+        },
+    },
+    tablet: {
+        name: "tablet",
+        styles: {
+            width: "640px",
+            height: "960px",
+        },
+    },
+    desktop: {
+        name: "desktop",
+        styles: {
+            width: "1024px",
+            height: "768px",
+        },
+    },
+};
+
+export default {
+    title: "Modal/Building Blocks/ModalDialog",
+    component: ModalDialog,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        viewport: {
+            viewports: customViewports,
+            defaultViewport: "desktop",
+        },
+        chromatic: {
+            viewports: [320, 640, 1024],
+        },
+    },
+    // Make the following props null in the control panel
+    // because setting an object for a React node is
+    // not practical to do manually.
+    argTypes: {
+        children: {
+            control: {type: null},
+        },
+        above: {
+            control: {type: null},
+        },
+        below: {
+            control: {type: null},
+        },
+    },
+};
+
+export const Default: StoryComponentType = (args) => (
+    <View style={styles.previewSizer}>
+        <View style={styles.modalPositioner}>
+            <ModalDialog {...args}>
```

Comment:
**suggestion:** What do you think about adding `aria-labelledby` referencing to a title within the `ModalPanel` contents? 

That could be a good way to show the a11y case presented in the component description and we could show engineers how to use custom Dialogs while preserving accessibility.

<img width="865" alt="Screen Shot 2022-08-26 at 8 52 28 AM" src="https://user-images.githubusercontent.com/843075/186907775-a9cb1daa-5dea-4344-b0be-373056f66064.png">

Reply:
Great idea!